interfaces/udev/backend.go

```diff
@@ -91,22 +92,46 @@ func ensureDirState(dir, glob string, content map[string]*osutil.FileState, snap
 	return errReload
 }
 
+func generateHash(bytes []byte) uint32 {
+	h := fnv.New32a()
```

Comment:
This hash isn't strong enough to handle uniqueness as intended here.

I suggest going with a simpler implementation:
```
var unique map[string][]byte
...
for _, snippet := range ... {
        unique[string(snippet)] = snippet
}
```

No need to test for containment. At the end of the process unique will have all the snippets in it.